.github/workflows/prepare_deployment_branch.yaml

```diff
@@ -4,7 +4,7 @@ on:
   schedule:
     # At 16:00 on Monday and Wednesday (https://crontab.guru/#0_16_*_*_1,3)
     # GitHub actions run in UTC (and EDT is UTC-4)
-    - cron: "0 16 * * 1,3"
+    - cron: "0 23 * * 1,3"
```

Comment:
Can you update the comment above to reflect the new time?

Reply:
good catch! Done